torq/umbrel-app.yml

```diff
@@ -51,28 +51,24 @@ gallery:
   - 4.jpg
   - 5.jpg
 releaseNotes: >-
-  A lot of improvements have been made in this release. It's hard to cover it all but here are some highlights:
-  
-    - Add advanced outputs table (UTXO management), with the ability to lock/unlock individual outputs.
-    - LND v0.18 support (+ inbound fees)
-    - CLN v24.05 support
-    - Tighter CLN integration via Torq plugin
-    - Fix opening a channel with CLN (missing close-to address)
-    - Fix CLN+LND on-chain transaction import (this will reimport on-chain transactions)
-    - Torq will import transactions + transaction inputs and outputs
-    - We advise to configure bitcoind connection details for extra annotation
-    - Ability to integrate Electrum (for unconfirmed on-chain notification support)
-    - System/workflow variables/templates in workflows
-    - example 1: proportionate fee template: 2000 - ({{Channels.peerGauge}} * 2000)
-    - example 2: dynamic max HTLC amount: {{Channels.localBalance}} / 2
-    - Channel open interceptor workflow (LND and CLN)
-    - Email is now a notification type in workflows (using SMTP)
-    - Ability to trigger a workflow via gRPC (and specify workflow/system variables)
-    - Manual and automatic emergency recovery file / SCB (for CLN and LND)
-    - Automatic includes: email, Github, dropbox, gdrive
-    - Beta feature: (Unauthenticated) Implementation agnostic Torq gRPC
-    - With methods: GetActiveNodes, GetNodeInformation, ConnectPeer, DisconnectPeer, OpenChannel, CloseChannel, SendPayment, CreateInvoice, CreateAddress, TagPeer, TriggerWorkflow, GetOnChainTransactionsByAddress, GetOnChainFeeEstimates, WaitInvoiceSettledByPaymentHash, SubscribeInvoiceStatuses, WaitOnChainTransactionsByAddress, SubscribeOnChainTransactions
-    - Package upgrades
+  v2.1 release notes:
```

Comment:
You could just replace this with something like `Highlights:`, since the umbrelOS UI will already render "What's new" and the version:

<img width="673" alt="image" src="https://github.com/user-attachments/assets/6cf26229-23ee-4746-8b29-543b813a12d0">

```suggestion
  Highlights:
```